### PR TITLE
Upgrade jawn 0.14.3 & tectonic

### PR DESCRIPTION
**Pull request by rintcius**
[ch9979]

### CODE REVIEW

**Comment by rintcius** [comment]
published snapshot 1.0.5-405db8a

File: project/build.sbt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-libraryDependencies += "org.kohsuke" % "github-api" % "1.95" exclude("org.jenkins-ci", "annotation-indexer")
+libraryDependencies += "org.kohsuke" % "github-api" % "1.101" exclude("org.jenkins-ci", "annotation-indexer")
```
Comment (alissapajer) [inline project/build.sbt]: I wonder if we still need this here. Anyone recall what it was for?

Comment (rintcius) [reply project/build.sbt]: Indeed, I was wondering that too. I saw it in some other place as well.

Comment (alissapajer) [reply project/build.sbt]: If we don't figure out what this was for, we could remove it and see what happens. I don't want to add more changes than necessary to the big fs2 bump though, so we could do that after all the fs2 PRs land.

**Pull request merged**